src/codecovio.jl

```diff
@@ -137,7 +137,13 @@ module Codecov
                 job          = ENV["BUILD_DEFINITIONNAME"],
                 slug         = ENV["BUILD_REPOSITORY_NAME"],
                 build        = ENV["BUILD_BUILDID"],
-            )            
+            )
+        elseif haskey(ENV, "GITHUB_ACTION") # GitHub Actions
+            kwargs = set_defaults(kwargs,
+                service      = "custom",
+                commit       = ENV["GITHUB_SHA"],
+                slug         = ENV["GITHUB_REPOSITORY"],
+            )
```

Comment:
add `branch = GITHUB_REF?` and maybe `pr?? = GITHUB_EVENT_NAME`?

Reply:
I think I had the `branch=` story in there before, and for some reason removed it, I think maybe it didn't work? I can't remember... 

I don't have time right now to make more changes and test them, so I would suggest we merge the current version (because that certainly works), and then if someone else has more time to add more info and test out whether that works we can do it in a follow up PR?